yt/data_objects/particle_trajectories.py

```diff
@@ -69,7 +67,6 @@ def __init__(
 
         if fields is None:
             fields = []
-        fields = list(OrderedDict.fromkeys(fields))
```

Comment:
This may not look obvious but the current implementation here is actually doing _nothing_, so I went with the option of preserving behaviour rather than intent. I think the original intent was to _sort_ the keys, in which case it should be simply reimplemented as
```python
fields = sorted(keys)
```
@jzuhone , do you remember this ? :)

Reply:
I think this is fine. I also think that someone came in here and did this later, because I don't recognize the code to be honest.

Reply:
Cool, thanks for your prompt answer